### ISSUE TITLE
CT unsupported fraction value

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/mean-share-and-balance#202]
```
mean-share-and-balance : fuzz : unbuilt : listenerOrderRandom
http://128.138.93.172/continuous-testing/ct-snapshots/1712827021364/mean-share-and-balance/mean-share-and-balance_en.html?continuousTest=%7B%22test%22%3A%5B%22mean-share-and-balance%22%2C%22fuzz%22%2C%22unbuilt%22%2C%22listenerOrderRandom%22%5D%2C%22snapshotName%22%3A%22snapshot-1712827021364%22%2C%22timestamp%22%3A1712827642258%7D&brand=phet&ea&fuzz&listenerOrder=random
Query: brand=phet&ea&fuzz&listenerOrder=random
Uncaught Error: Assertion failed: unsupported fraction value
Error: Assertion failed: unsupported fraction value
at window.assertions.assertFunction (http://128.138.93.172/continuous-testing/ct-snapshots/1712827021364/assert/js/assert.js:28:13)
at assert (NotepadAppleNode.ts:65:18)
at listener (TinyEmitter.ts:213:6)
at notifyLoop (TinyEmitter.ts:185:17)
at emit (ReadOnlyProperty.ts:352:22)
at _notifyListeners (ReadOnlyProperty.ts:296:13)
at unguardedSet (ReadOnlyProperty.ts:276:11)
at set (Property.ts:54:10)
at (FairShareModel.ts:530:38)
at addSnackAddedListener (FairShareModel.ts:126:13)
[URL] http://128.138.93.172/continuous-testing/aqua/html/sim-test.html?url=..%2F..%2Fct-snapshots%2F1712827021364%2Fmean-share-and-balance%2Fmean-share-and-balance_en.html&simQueryParameters=brand%3Dphet%26ea%26fuzz%26listenerOrder%3Drandom&testInfo=%7B%22test%22%3A%5B%22mean-share-and-balance%22%2C%22fuzz%22%2C%22unbuilt%22%2C%22listenerOrderRandom%22%5D%2C%22snapshotName%22%3A%22snapshot-1712827021364%22%2C%22timestamp%22%3A1712827642258%7D
[NAVIGATED] http://128.138.93.172/continuous-testing/aqua/html/sim-test.html?url=..%2F..%2Fct-snapshots%2F1712827021364%2Fmean-share-and-balance%2Fmean-share-and-balance_en.html&simQueryParameters=brand%3Dphet%26ea%26fuzz%26listenerOrder%3Drandom&testInfo=%7B%22test%22%3A%5B%22mean-share-and-balance%22%2C%22fuzz%22%2C%22unbuilt%22%2C%22listenerOrderRandom%22%5D%2C%22snapshotName%22%3A%22snapshot-1712827021364%22%2C%22timestamp%22%3A1712827642258%7D
[ATTACHED]
[NAVIGATED] about:blank
[NAVIGATED] http://128.138.93.172/continuous-testing/ct-snapshots/1712827021364/mean-share-and-balance/mean-share-and-balance_en.html?continuousTest=%7B%22test%22%3A%5B%22mean-share-and-balance%22%2C%22fuzz%22%2C%22unbuilt%22%2C%22listenerOrderRandom%22%5D%2C%22snapshotName%22%3A%22snapshot-1712827021364%22%2C%22timestamp%22%3A1712827642258%7D&brand=phet&ea&fuzz&listenerOrder=random
[CONSOLE] enabling assert
[CONSOLE] listenerOrder random seed: 977609
[CONSOLE] continuous-test-load
[CONSOLE] Assertion failed: unsupported fraction value
[PAGE ERROR] Error: Error: Assertion failed: unsupported fraction value
at window.assertions.assertFunction (http://128.138.93.172/continuous-testing/ct-snapshots/1712827021364/assert/js/assert.js:28:13)
at http://128.138.93.172/continuous-testing/ct-snapshots/1712827021364/chipper/dist/js/mean-share-and-balance/js/fair-share/view/NotepadAppleNode.js:50:19
at TinyProperty.notifyLoop (http://128.138.93.172/continuous-testing/ct-snapshots/1712827021364/chipper/dist/js/axon/js/TinyEmitter.js:176:7)
at TinyProperty.emit (http://128.138.93.172/continuous-testing/ct-snapshots/1712827021364/chipper/dist/js/axon/js/TinyEmitter.js:154:18)
at Property._notifyListeners (http://128.138.93.172/continuous-testing/ct-snapshots/1712827021364/chipper/dist/js/axon/js/ReadOnlyProperty.js:264:23)
at Property.unguardedSet (http://128.138.93.172/continuous-testing/ct-snapshots/1712827021364/chipper/dist/js/axon/js/ReadOnlyProperty.js:211:14)
at Property.set (http://128.138.93.172/continuous-testing/ct-snapshots/1712827021364/chipper/dist/js/axon/js/ReadOnlyProperty.js:193:12)
at set value [as value] (http://128.138.93.172/continuous-testing/ct-snapshots/1712827021364/chipper/dist/js/axon/js/Property.js:48:11)
at FairShareModel.addSnackAddedListener (http://128.138.93.172/continuous-testing/ct-snapshots/1712827021364/chipper/dist/js/mean-share-and-balance/js/fair-share/model/FairShareModel.js:413:40)
at http://128.138.93.172/continuous-testing/ct-snapshots/1712827021364/chipper/dist/js/mean-share-and-balance/js/fair-share/model/FairShareModel.js:102:14
[CONSOLE] continuous-test-error
[CONSOLE] Assertion failed: reentry detected, value=2, oldValue=3
[PAGE ERROR] Error: Error: Assertion failed: reentry detected, value=2, oldValue=3
at window.assertions.assertFunction (http://128.138.93.172/continuous-testing/ct-snapshots/1712827021364/assert/js/assert.js:28:13)
at DerivedProperty._notifyListeners (http://128.138.93.172/continuous-testing/ct-snapshots/1712827021364/chipper/dist/js/axon/js/ReadOnlyProperty.js:262:15)
at DerivedProperty.unguardedSet (http://128.138.93.172/continuous-testing/ct-snapshots/1712827021364/chipper/dist/js/axon/js/ReadOnlyProperty.js:211:14)
at DerivedProperty.set (http://128.138.93.172/continuous-testing/ct-snapshots/1712827021364/chipper/dist/js/axon/js/ReadOnlyProperty.js:193:12)
at DerivedProperty.getDerivedPropertyListener (http://128.138.93.172/continuous-testing/ct-snapshots/1712827021364/chipper/dist/js/axon/js/DerivedProperty.js:121:13)
at TinyProperty.notifyLoop (http://128.138.93.172/continuous-testing/ct-snapshots/1712827021364/chipper/dist/js/axon/js/TinyEmitter.js:176:7)
at TinyProperty.emit (http://128.138.93.172/continuous-testing/ct-snapshots/1712827021364/chipper/dist/js/axon/js/TinyEmitter.js:154:18)
at NumberProperty._notifyListeners (http://128.138.93.172/continuous-testing/ct-snapshots/1712827021364/chipper/dist/js/axon/js/ReadOnlyProperty.js:264:23)
at NumberProperty.unguardedSet (http://128.138.93.172/continuous-testing/ct-snapshots/1712827021364/chipper/dist/js/axon/js/ReadOnlyProperty.js:211:14)
at NumberProperty.set (http://128.138.93.172/continuous-testing/ct-snapshots/1712827021364/chipper/dist/js/axon/js/ReadOnlyProperty.js:193:12)
[CONSOLE] continuous-test-error
[CONSOLE] Assertion failed: reentry detected, value=1/1, oldValue=0/3
[PAGE ERROR] Error: Error: Assertion failed: reentry detected, value=1/1, oldValue=0/3
at window.assertions.assertFunction (http://128.138.93.172/continuous-testing/ct-snapshots/1712827021364/assert/js/assert.js:28:13)
at Property._notifyListeners (http://128.138.93.172/continuous-testing/ct-snapshots/1712827021364/chipper/dist/js/axon/js/ReadOnlyProperty.js:262:15)
at Property.unguardedSet (http://128.138.93.172/continuous-testing/ct-snapshots/1712827021364/chipper/dist/js/axon/js/ReadOnlyProperty.js:211:14)
at Property.set (http://128.138.93.172/continuous-testing/ct-snapshots/1712827021364/chipper/dist/js/axon/js/ReadOnlyProperty.js:193:12)
at set value [as value] (http://128.138.93.172/continuous-testing/ct-snapshots/1712827021364/chipper/dist/js/axon/js/Property.js:48:11)
at http://128.138.93.172/continuous-testing/ct-snapshots/1712827021364/chipper/dist/js/mean-share-and-balance/js/fair-share/model/FairShareModel.js:204:40
at Array.forEach (<anonymous>)
at handleModeChange (http://128.138.93.172/continuous-testing/ct-snapshots/1712827021364/chipper/dist/js/mean-share-and-balance/js/fair-share/model/FairShareModel.js:203:29)
at TinyProperty.notifyLoop (http://128.138.93.172/continuous-testing/ct-snapshots/1712827021364/chipper/dist/js/axon/js/TinyEmitter.js:176:7)
at TinyProperty.emit (http://128.138.93.172/continuous-testing/ct-snapshots/1712827021364/chipper/dist/js/axon/js/TinyEmitter.js:154:18)
[CONSOLE] continuous-test-error
[CONSOLE] Assertion failed: reentry detected, value=2, oldValue=3
[PAGE ERROR] Error: Error: Assertion failed: reentry detected, value=2, oldValue=3
at window.assertions.assertFunction (http://128.138.93.172/continuous-testing/ct-snapshots/1712827021364/assert/js/assert.js:28:13)
at NumberProperty._notifyListeners (http://128.138.93.172/continuous-testing/ct-snapshots/1712827021364/chipper/dist/js/axon/js/ReadOnlyProperty.js:262:15)
at http://128.138.93.172/continuous-testing/ct-snapshots/1712827021364/chipper/dist/js/axon/js/ReadOnlyProperty.js:309:47
at NumberProperty.setValueAndRange (http://128.138.93.172/continuous-testing/ct-snapshots/1712827021364/chipper/dist/js/axon/js/NumberProperty.js:159:29)
at NumberProperty.resetValueAndRange (http://128.138.93.172/continuous-testing/ct-snapshots/1712827021364/chipper/dist/js/axon/js/NumberProperty.js:168:10)
at NumberProperty.reset (http://128.138.93.172/continuous-testing/ct-snapshots/1712827021364/chipper/dist/js/axon/js/NumberProperty.js:133:10)
at FairShareModel.reset (http://128.138.93.172/continuous-testing/ct-snapshots/1712827021364/chipper/dist/js/mean-share-and-balance/js/common/model/SharingModel.js:183:33)
at FairShareModel.reset (http://128.138.93.172/continuous-testing/ct-snapshots/1712827021364/chipper/dist/js/mean-share-and-balance/js/fair-share/model/FairShareModel.js:427:11)
at listener (http://128.138.93.172/continuous-testing/ct-snapshots/1712827021364/chipper/dist/js/mean-share-and-balance/js/common/view/MeanShareAndBalanceScreenView.js:32:15)
at TinyEmitter.notifyLoop (http://128.138.93.172/continuous-testing/ct-snapshots/1712827021364/chipper/dist/js/axon/js/TinyEmitter.js:176:7)
[CONSOLE] continuous-test-error

id: "Sparky Node Puppeteer"
Snapshot from 4/11/2024, 3:17:01 AM

----------------------------------

mean-share-and-balance : fuzz : unbuilt : listenerOrderRandom
http://128.138.93.172/continuous-testing/ct-snapshots/1712827021364/mean-share-and-balance/mean-share-and-balance_en.html?continuousTest=%7B%22test%22%3A%5B%22mean-share-and-balance%22%2C%22fuzz%22%2C%22unbuilt%22%2C%22listenerOrderRandom%22%5D%2C%22snapshotName%22%3A%22snapshot-1712827021364%22%2C%22timestamp%22%3A1712842952545%7D&brand=phet&ea&fuzz&listenerOrder=random
Query: brand=phet&ea&fuzz&listenerOrder=random
Uncaught Error: Assertion failed: unsupported fraction value
Error: Assertion failed: unsupported fraction value
at window.assertions.assertFunction (http://128.138.93.172/continuous-testing/ct-snapshots/1712827021364/assert/js/assert.js:28:13)
at assert (NotepadAppleNode.ts:65:18)
at listener (TinyEmitter.ts:213:6)
at notifyLoop (TinyEmitter.ts:185:17)
at emit (ReadOnlyProperty.ts:352:22)
at _notifyListeners (ReadOnlyProperty.ts:296:13)
at unguardedSet (ReadOnlyProperty.ts:276:11)
at set (Property.ts:54:10)
at (FairShareModel.ts:530:38)
at addSnackAddedListener (FairShareModel.ts:126:13)
[URL] http://128.138.93.172/continuous-testing/aqua/html/sim-test.html?url=..%2F..%2Fct-snapshots%2F1712827021364%2Fmean-share-and-balance%2Fmean-share-and-balance_en.html&simQueryParameters=brand%3Dphet%26ea%26fuzz%26listenerOrder%3Drandom&testInfo=%7B%22test%22%3A%5B%22mean-share-and-balance%22%2C%22fuzz%22%2C%22unbuilt%22%2C%22listenerOrderRandom%22%5D%2C%22snapshotName%22%3A%22snapshot-1712827021364%22%2C%22timestamp%22%3A1712842952545%7D
[NAVIGATED] http://128.138.93.172/continuous-testing/aqua/html/sim-test.html?url=..%2F..%2Fct-snapshots%2F1712827021364%2Fmean-share-and-balance%2Fmean-share-and-balance_en.html&simQueryParameters=brand%3Dphet%26ea%26fuzz%26listenerOrder%3Drandom&testInfo=%7B%22test%22%3A%5B%22mean-share-and-balance%22%2C%22fuzz%22%2C%22unbuilt%22%2C%22listenerOrderRandom%22%5D%2C%22snapshotName%22%3A%22snapshot-1712827021364%22%2C%22timestamp%22%3A1712842952545%7D
[ATTACHED]
[NAVIGATED] about:blank
[NAVIGATED] http://128.138.93.172/continuous-testing/ct-snapshots/1712827021364/mean-share-and-balance/mean-share-and-balance_en.html?continuousTest=%7B%22test%22%3A%5B%22mean-share-and-balance%22%2C%22fuzz%22%2C%22unbuilt%22%2C%22listenerOrderRandom%22%5D%2C%22snapshotName%22%3A%22snapshot-1712827021364%22%2C%22timestamp%22%3A1712842952545%7D&brand=phet&ea&fuzz&listenerOrder=random
[CONSOLE] enabling assert
[CONSOLE] listenerOrder random seed: 419273
[CONSOLE] continuous-test-load
[CONSOLE] Assertion failed: unsupported fraction value
[PAGE ERROR] Error: Error: Assertion failed: unsupported fraction value
at window.assertions.assertFunction (http://128.138.93.172/continuous-testing/ct-snapshots/1712827021364/assert/js/assert.js:28:13)
at http://128.138.93.172/continuous-testing/ct-snapshots/1712827021364/chipper/dist/js/mean-share-and-balance/js/fair-share/view/NotepadAppleNode.js:50:19
at TinyProperty.notifyLoop (http://128.138.93.172/continuous-testing/ct-snapshots/1712827021364/chipper/dist/js/axon/js/TinyEmitter.js:176:7)
at TinyProperty.emit (http://128.138.93.172/continuous-testing/ct-snapshots/1712827021364/chipper/dist/js/axon/js/TinyEmitter.js:154:18)
at Property._notifyListeners (http://128.138.93.172/continuous-testing/ct-snapshots/1712827021364/chipper/dist/js/axon/js/ReadOnlyProperty.js:264:23)
at Property.unguardedSet (http://128.138.93.172/continuous-testing/ct-snapshots/1712827021364/chipper/dist/js/axon/js/ReadOnlyProperty.js:211:14)
at Property.set (http://128.138.93.172/continuous-testing/ct-snapshots/1712827021364/chipper/dist/js/axon/js/ReadOnlyProperty.js:193:12)
at set value [as value] (http://128.138.93.172/continuous-testing/ct-snapshots/1712827021364/chipper/dist/js/axon/js/Property.js:48:11)
at FairShareModel.addSnackAddedListener (http://128.138.93.172/continuous-testing/ct-snapshots/1712827021364/chipper/dist/js/mean-share-and-balance/js/fair-share/model/FairShareModel.js:413:40)
at http://128.138.93.172/continuous-testing/ct-snapshots/1712827021364/chipper/dist/js/mean-share-and-balance/js/fair-share/model/FairShareModel.js:102:14
[CONSOLE] continuous-test-error

id: "Sparky Node Puppeteer"
Snapshot from 4/11/2024, 3:17:01 AM

----------------------------------

mean-share-and-balance : multitouch-fuzz : unbuilt
http://128.138.93.172/continuous-testing/ct-snapshots/1712827021364/mean-share-and-balance/mean-share-and-balance_en.html?continuousTest=%7B%22test%22%3A%5B%22mean-share-and-balance%22%2C%22multitouch-fuzz%22%2C%22unbuilt%22%5D%2C%22snapshotName%22%3A%22snapshot-1712827021364%22%2C%22timestamp%22%3A1712832787638%7D&brand=phet&ea&fuzz&fuzzPointers=2&supportsPanAndZoom=false
Query: brand=phet&ea&fuzz&fuzzPointers=2&supportsPanAndZoom=false
Uncaught Error: Assertion failed: Every plate should have at least one apple unless the total number of snacks is 0.
Error: Assertion failed: Every plate should have at least one apple unless the total number of snacks is 0.
at window.assertions.assertFunction (http://128.138.93.172/continuous-testing/ct-snapshots/1712827021364/assert/js/assert.js:28:13)
at assert (FairShareModel.ts:291:22)
at Array.forEach
at forEach (FairShareModel.ts:277:31)
at listener (TinyEmitter.ts:213:6)
at notifyLoop (TinyEmitter.ts:185:17)
at emit (ReadOnlyProperty.ts:352:22)
at _notifyListeners (ReadOnlyProperty.ts:296:13)
at unguardedSet (ReadOnlyProperty.ts:276:11)
at set (Property.ts:65:10)
[URL] http://128.138.93.172/continuous-testing/aqua/html/sim-test.html?url=..%2F..%2Fct-snapshots%2F1712827021364%2Fmean-share-and-balance%2Fmean-share-and-balance_en.html&simQueryParameters=brand%3Dphet%26ea%26fuzz%26fuzzPointers%3D2%26supportsPanAndZoom%3Dfalse&testInfo=%7B%22test%22%3A%5B%22mean-share-and-balance%22%2C%22multitouch-fuzz%22%2C%22unbuilt%22%5D%2C%22snapshotName%22%3A%22snapshot-1712827021364%22%2C%22timestamp%22%3A1712832787638%7D
[NAVIGATED] http://128.138.93.172/continuous-testing/aqua/html/sim-test.html?url=..%2F..%2Fct-snapshots%2F1712827021364%2Fmean-share-and-balance%2Fmean-share-and-balance_en.html&simQueryParameters=brand%3Dphet%26ea%26fuzz%26fuzzPointers%3D2%26supportsPanAndZoom%3Dfalse&testInfo=%7B%22test%22%3A%5B%22mean-share-and-balance%22%2C%22multitouch-fuzz%22%2C%22unbuilt%22%5D%2C%22snapshotName%22%3A%22snapshot-1712827021364%22%2C%22timestamp%22%3A1712832787638%7D
[ATTACHED]
[NAVIGATED] about:blank
[NAVIGATED] http://128.138.93.172/continuous-testing/ct-snapshots/1712827021364/mean-share-and-balance/mean-share-and-balance_en.html?continuousTest=%7B%22test%22%3A%5B%22mean-share-and-balance%22%2C%22multitouch-fuzz%22%2C%22unbuilt%22%5D%2C%22snapshotName%22%3A%22snapshot-1712827021364%22%2C%22timestamp%22%3A1712832787638%7D&brand=phet&ea&fuzz&fuzzPointers=2&supportsPanAndZoom=false
[CONSOLE] enabling assert
[CONSOLE] continuous-test-load
[CONSOLE] Assertion failed: Every plate should have at least one apple unless the total number of snacks is 0.
[PAGE ERROR] Error: Error: Assertion failed: Every plate should have at least one apple unless the total number of snacks is 0.
at window.assertions.assertFunction (http://128.138.93.172/continuous-testing/ct-snapshots/1712827021364/assert/js/assert.js:28:13)
at http://128.138.93.172/continuous-testing/ct-snapshots/1712827021364/chipper/dist/js/mean-share-and-balance/js/fair-share/model/FairShareModel.js:224:23
at Array.forEach (<anonymous>)
at handleModeChange (http://128.138.93.172/continuous-testing/ct-snapshots/1712827021364/chipper/dist/js/mean-share-and-balance/js/fair-share/model/FairShareModel.js:214:32)
at TinyProperty.notifyLoop (http://128.138.93.172/continuous-testing/ct-snapshots/1712827021364/chipper/dist/js/axon/js/TinyEmitter.js:176:7)
at TinyProperty.emit (http://128.138.93.172/continuous-testing/ct-snapshots/1712827021364/chipper/dist/js/axon/js/TinyEmitter.js:154:18)
at EnumerationProperty._notifyListeners (http://128.138.93.172/continuous-testing/ct-snapshots/1712827021364/chipper/dist/js/axon/js/ReadOnlyProperty.js:264:23)
at EnumerationProperty.unguardedSet (http://128.138.93.172/continuous-testing/ct-snapshots/1712827021364/chipper/dist/js/axon/js/ReadOnlyProperty.js:211:14)
at EnumerationProperty.set (http://128.138.93.172/continuous-testing/ct-snapshots/1712827021364/chipper/dist/js/axon/js/ReadOnlyProperty.js:193:12)
at EnumerationProperty.set (http://128.138.93.172/continuous-testing/ct-snapshots/1712827021364/chipper/dist/js/axon/js/Property.js:58:11)
[CONSOLE] continuous-test-error

id: "Sparky Node Puppeteer"
Snapshot from 4/11/2024, 3:17:01 AM

----------------------------------

mean-share-and-balance : pan-and-zoom-fuzz : unbuilt
http://128.138.93.172/continuous-testing/ct-snapshots/1712827021364/mean-share-and-balance/mean-share-and-balance_en.html?continuousTest=%7B%22test%22%3A%5B%22mean-share-and-balance%22%2C%22pan-and-zoom-fuzz%22%2C%22unbuilt%22%5D%2C%22snapshotName%22%3A%22snapshot-1712827021364%22%2C%22timestamp%22%3A1712843338715%7D&brand=phet&ea&fuzz&fuzzPointers=2&supportsPanAndZoom=true
Query: brand=phet&ea&fuzz&fuzzPointers=2&supportsPanAndZoom=true
Uncaught Error: Assertion failed: Every plate should have at least one apple unless the total number of snacks is 0.
Error: Assertion failed: Every plate should have at least one apple unless the total number of snacks is 0.
at window.assertions.assertFunction (http://128.138.93.172/continuous-testing/ct-snapshots/1712827021364/assert/js/assert.js:28:13)
at assert (FairShareModel.ts:291:22)
at Array.forEach
at forEach (FairShareModel.ts:277:31)
at listener (TinyEmitter.ts:213:6)
at notifyLoop (TinyEmitter.ts:185:17)
at emit (ReadOnlyProperty.ts:352:22)
at _notifyListeners (ReadOnlyProperty.ts:296:13)
at unguardedSet (ReadOnlyProperty.ts:276:11)
at set (Property.ts:65:10)
[URL] http://128.138.93.172/continuous-testing/aqua/html/sim-test.html?url=..%2F..%2Fct-snapshots%2F1712827021364%2Fmean-share-and-balance%2Fmean-share-and-balance_en.html&simQueryParameters=brand%3Dphet%26ea%26fuzz%26fuzzPointers%3D2%26supportsPanAndZoom%3Dtrue&testInfo=%7B%22test%22%3A%5B%22mean-share-and-balance%22%2C%22pan-and-zoom-fuzz%22%2C%22unbuilt%22%5D%2C%22snapshotName%22%3A%22snapshot-1712827021364%22%2C%22timestamp%22%3A1712843338715%7D
[NAVIGATED] http://128.138.93.172/continuous-testing/aqua/html/sim-test.html?url=..%2F..%2Fct-snapshots%2F1712827021364%2Fmean-share-and-balance%2Fmean-share-and-balance_en.html&simQueryParameters=brand%3Dphet%26ea%26fuzz%26fuzzPointers%3D2%26supportsPanAndZoom%3Dtrue&testInfo=%7B%22test%22%3A%5B%22mean-share-and-balance%22%2C%22pan-and-zoom-fuzz%22%2C%22unbuilt%22%5D%2C%22snapshotName%22%3A%22snapshot-1712827021364%22%2C%22timestamp%22%3A1712843338715%7D
[ATTACHED]
[NAVIGATED] about:blank
[NAVIGATED] http://128.138.93.172/continuous-testing/ct-snapshots/1712827021364/mean-share-and-balance/mean-share-and-balance_en.html?continuousTest=%7B%22test%22%3A%5B%22mean-share-and-balance%22%2C%22pan-and-zoom-fuzz%22%2C%22unbuilt%22%5D%2C%22snapshotName%22%3A%22snapshot-1712827021364%22%2C%22timestamp%22%3A1712843338715%7D&brand=phet&ea&fuzz&fuzzPointers=2&supportsPanAndZoom=true
[CONSOLE] enabling assert
[CONSOLE] continuous-test-load
[CONSOLE] Assertion failed: Every plate should have at least one apple unless the total number of snacks is 0.
[PAGE ERROR] Error: Error: Assertion failed: Every plate should have at least one apple unless the total number of snacks is 0.
at window.assertions.assertFunction (http://128.138.93.172/continuous-testing/ct-snapshots/1712827021364/assert/js/assert.js:28:13)
at http://128.138.93.172/continuous-testing/ct-snapshots/1712827021364/chipper/dist/js/mean-share-and-balance/js/fair-share/model/FairShareModel.js:224:23
at Array.forEach (<anonymous>)
at handleModeChange (http://128.138.93.172/continuous-testing/ct-snapshots/1712827021364/chipper/dist/js/mean-share-and-balance/js/fair-share/model/FairShareModel.js:214:32)
at TinyProperty.notifyLoop (http://128.138.93.172/continuous-testing/ct-snapshots/1712827021364/chipper/dist/js/axon/js/TinyEmitter.js:176:7)
at TinyProperty.emit (http://128.138.93.172/continuous-testing/ct-snapshots/1712827021364/chipper/dist/js/axon/js/TinyEmitter.js:154:18)
at EnumerationProperty._notifyListeners (http://128.138.93.172/continuous-testing/ct-snapshots/1712827021364/chipper/dist/js/axon/js/ReadOnlyProperty.js:264:23)
at EnumerationProperty.unguardedSet (http://128.138.93.172/continuous-testing/ct-snapshots/1712827021364/chipper/dist/js/axon/js/ReadOnlyProperty.js:211:14)
at EnumerationProperty.set (http://128.138.93.172/continuous-testing/ct-snapshots/1712827021364/chipper/dist/js/axon/js/ReadOnlyProperty.js:193:12)
at EnumerationProperty.set (http://128.138.93.172/continuous-testing/ct-snapshots/1712827021364/chipper/dist/js/axon/js/Property.js:58:11)
[CONSOLE] continuous-test-error

id: "Sparky Node Puppeteer"
Snapshot from 4/11/2024, 3:17:01 AM

----------------------------------

mean-share-and-balance : phet-io-fuzz : unbuilt
http://128.138.93.172/continuous-testing/ct-snapshots/1712827021364/mean-share-and-balance/mean-share-and-balance_en.html?continuousTest=%7B%22test%22%3A%5B%22mean-share-and-balance%22%2C%22phet-io-fuzz%22%2C%22unbuilt%22%5D%2C%22snapshotName%22%3A%22snapshot-1712827021364%22%2C%22timestamp%22%3A1712843028128%7D&ea&brand=phet-io&phetioStandalone&fuzz
Query: ea&brand=phet-io&phetioStandalone&fuzz
Uncaught Error: Assertion failed: Every plate should have at least one apple unless the total number of snacks is 0.
Error: Assertion failed: Every plate should have at least one apple unless the total number of snacks is 0.
at window.assertions.assertFunction (http://128.138.93.172/continuous-testing/ct-snapshots/1712827021364/assert/js/assert.js:28:13)
at assert (FairShareModel.ts:291:22)
at Array.forEach
at forEach (FairShareModel.ts:277:31)
at listener (TinyEmitter.ts:213:6)
at notifyLoop (TinyEmitter.ts:185:17)
at emit (ReadOnlyProperty.ts:352:22)
at _notifyListeners (ReadOnlyProperty.ts:296:13)
at unguardedSet (ReadOnlyProperty.ts:276:11)
at set (Property.ts:65:10)
[URL] http://128.138.93.172/continuous-testing/aqua/html/sim-test.html?url=..%2F..%2Fct-snapshots%2F1712827021364%2Fmean-share-and-balance%2Fmean-share-and-balance_en.html&simQueryParameters=ea%26brand%3Dphet-io%26phetioStandalone%26fuzz&testInfo=%7B%22test%22%3A%5B%22mean-share-and-balance%22%2C%22phet-io-fuzz%22%2C%22unbuilt%22%5D%2C%22snapshotName%22%3A%22snapshot-1712827021364%22%2C%22timestamp%22%3A1712843028128%7D
[NAVIGATED] http://128.138.93.172/continuous-testing/aqua/html/sim-test.html?url=..%2F..%2Fct-snapshots%2F1712827021364%2Fmean-share-and-balance%2Fmean-share-and-balance_en.html&simQueryParameters=ea%26brand%3Dphet-io%26phetioStandalone%26fuzz&testInfo=%7B%22test%22%3A%5B%22mean-share-and-balance%22%2C%22phet-io-fuzz%22%2C%22unbuilt%22%5D%2C%22snapshotName%22%3A%22snapshot-1712827021364%22%2C%22timestamp%22%3A1712843028128%7D
[ATTACHED]
[NAVIGATED] about:blank
[NAVIGATED] http://128.138.93.172/continuous-testing/ct-snapshots/1712827021364/mean-share-and-balance/mean-share-and-balance_en.html?continuousTest=%7B%22test%22%3A%5B%22mean-share-and-balance%22%2C%22phet-io-fuzz%22%2C%22unbuilt%22%5D%2C%22snapshotName%22%3A%22snapshot-1712827021364%22%2C%22timestamp%22%3A1712843028128%7D&ea&brand=phet-io&phetioStandalone&fuzz
[CONSOLE] enabling assert
[CONSOLE] continuous-test-load
[CONSOLE] Assertion failed: Every plate should have at least one apple unless the total number of snacks is 0.
[PAGE ERROR] Error: Error: Assertion failed: Every plate should have at least one apple unless the total number of snacks is 0.
at window.assertions.assertFunction (http://128.138.93.172/continuous-testing/ct-snapshots/1712827021364/assert/js/assert.js:28:13)
at http://128.138.93.172/continuous-testing/ct-snapshots/1712827021364/chipper/dist/js/mean-share-and-balance/js/fair-share/model/FairShareModel.js:224:23
at Array.forEach (<anonymous>)
at handleModeChange (http://128.138.93.172/continuous-testing/ct-snapshots/1712827021364/chipper/dist/js/mean-share-and-balance/js/fair-share/model/FairShareModel.js:214:32)
at TinyProperty.notifyLoop (http://128.138.93.172/continuous-testing/ct-snapshots/1712827021364/chipper/dist/js/axon/js/TinyEmitter.js:176:7)
at TinyProperty.emit (http://128.138.93.172/continuous-testing/ct-snapshots/1712827021364/chipper/dist/js/axon/js/TinyEmitter.js:154:18)
at EnumerationProperty._notifyListeners (http://128.138.93.172/continuous-testing/ct-snapshots/1712827021364/chipper/dist/js/axon/js/ReadOnlyProperty.js:264:23)
at EnumerationProperty.unguardedSet (http://128.138.93.172/continuous-testing/ct-snapshots/1712827021364/chipper/dist/js/axon/js/ReadOnlyProperty.js:211:14)
at EnumerationProperty.set (http://128.138.93.172/continuous-testing/ct-snapshots/1712827021364/chipper/dist/js/axon/js/ReadOnlyProperty.js:193:12)
at EnumerationProperty.set (http://128.138.93.172/continuous-testing/ct-snapshots/1712827021364/chipper/dist/js/axon/js/Property.js:58:11)
[CONSOLE] continuous-test-error

id: "Sparky Node Puppeteer"
Snapshot from 4/11/2024, 3:17:01 AM
```

**Comment by jbphet** [comment]
@marlitas and I looked at this together and found a possible fix.  The animation in `Snack.ts` was sometimes being started before the associated Property value was being set.  In cases where the snack was already at the destination position, this could cause the handlers for the animation finishing to be invoked immediately, and the Property being set afterwards.  This would cause a problematic state.  We adjusted the order and re-red the fuzz testing locally and weren't seeing the problem.  I've committed the change, and we will keep an eye on CT to see if the problem is really addressed.
